<ui version="4.0" >
 <class>TopicChooserDialog</class>
 <widget class="QDialog" name="TopicChooserDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>373</width>
    <height>224</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Choose Topic</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <layout class="QVBoxLayout" name="verticalLayout" >
     <item>
      <widget class="QLabel" name="topicLabel" >
       <property name="text" >
        <string>Choose a topic for </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listWidget" />
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" >
       <item>
        <spacer name="horizontalSpacer" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>208</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="displayButton" >
         <property name="text" >
          <string>Display</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton" >
         <property name="text" >
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>TopicChooserDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>315</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel" >
     <x>298</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
